### PR TITLE
[TKAI-3107] Fetch bounties from `bepro api`

### PR DESCRIPTION
**Pull request by RTAndrew**
- Only show opened bounties
- Only show up to 4 bounties

image

### CODE REVIEW

File: lib/fetch-api/index.ts
Comment (ickas) [inline lib/fetch-api/index.ts:1]: @RTAndrew I'm getting this error 👀

Screen Shot 2023-03-23 at 09 52 18

File: pages/index.tsx [hunk 1]
```diff
@@ -54,12 +56,15 @@ const Home: NextPage = () => {
...
 
+	const { data } = await getRequest<IBeproApi>(`/search/issues`);
+
 	await apolloClient.query({
 		query: STORIES_QUERY,
 	});
 
 	return {
 		props: {
+			bounties: data?.rows.filter((r) => r.state === State.Open).slice(0, 4),
```
Comment (ickas) [inline pages/index.tsx]: We can show the most recent open bounties, and if we don't have four, we can also show the most recent close. What do you think?

**Review by ickas — COMMENTED** [review]
Please indent using spaces instead tabs 🙏
We can add a `.prettierrc.json` or something.

```
{
  "tabWidth": 2,
  "useTabs": false,
}
```

File: pages/index.tsx (hunk 1 above)
Comment (RTAndrew) [reply pages/index.tsx]: I think that's a great idea, as there are not many bounties.

**Review by ickas — APPROVED** [review]

**Pull request merged**